src/RoundTabler/PerformanceResult.java

```diff
@@ -0,0 +1,48 @@
+package RoundTabler;
+
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+
+
+public class PerformanceResult{
+
+	public String TableName;
+	public String TableColumn;
+	public int RowsScanned;
+	public int RowsMatched;
+	public String MatchType;    //"PCIDSS or NACHA"
```

Comment:
this can be accessed from the config if wanted.

Reply:
@zachrussell12  Each performance result will be for each type of scan for a table row and column type.  And if a user runs the tool with both, we need to be able to report MatchTypes for both PCI and NACHA on separate lines/outputs of performance results.

Reply:
I see I misunderstood. I thought you were referencing something similar to the --type parameter where the user specifies if they want to do an ALL, just PCI, or just NACHA scan. I understand now.